### PR TITLE
fix(k8smeta): introduce /proc scan to recover the pod uid of threads already spawned

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind bug

/kind feature

**Any specific area of the project related to this PR?**

/area plugins

**What this PR does / why we need it**:

This PR introduces a proc-scan feature in the k8smeta plugin init phase. We need this to collect the pod uid of threads already running when Falco and the k8smeta plugin are started. 

**Which issue(s) this PR fixes**:

Fixes #https://github.com/falcosecurity/plugins/issues/514

**Special notes for your reviewer**:

### CODE REVIEW

**Review by alacuku — APPROVED** [review]
/lgtm

**Comment by alacuku** [comment]
Thanks for the fix @Andreagit97!

File: plugins/k8smeta/src/plugin.cpp [hunk 1]
```diff
@@ -217,6 +256,89 @@ void my_plugin::parse_init_config(nlohmann::json& config_json)
...
+                if(!pod_uid.empty())
+                {
+                    m_thread_id_pod_uid_map[tid] = pod_uid;
+                    SPDLOG_TRACE("Found thread with tid '{}' and pod uid '{}'",
+                                 tid, pod_uid);
+                }
+            }
+            else
+            {
+                SPDLOG_WARN("cannot retrieve the cgroup first line for '{}'",
```
Comment (Andreagit97) [inline plugins/k8smeta/src/plugin.cpp:329]: it would be great to add the error here

File: plugins/k8smeta/src/plugin.cpp [hunk 2]
```diff
@@ -217,6 +256,89 @@ void my_plugin::parse_init_config(nlohmann::json& config_json)
...
+                            proc_path);
+            }
+            file.close();
+        }
+        else
+        {
+            SPDLOG_WARN("cannot open '{}'. Skip it.", proc_path);
```
Comment (Andreagit97) [inline plugins/k8smeta/src/plugin.cpp:336]: and also here

File: plugins/k8smeta/src/shared_with_tests_consts.h [hunk 3]
```diff
@@ -77,7 +77,7 @@ limitations under the License.
 // Generic plugin consts
 /////////////////////////
 #define PLUGIN_NAME "k8smeta"
-#define PLUGIN_VERSION "0.1.0"
+#define PLUGIN_VERSION "0.1.1"
```
Comment (Andreagit97) [inline plugins/k8smeta/src/shared_with_tests_consts.h:80]: We bumped a patch version of the plugin

**Comment by alacuku** [comment]
hey @Andreagit97, is this PR ready?

**Comment by Andreagit97** [comment]
yes! i removed the [WIP]

**Review by alacuku — APPROVED** [review]
/lgtm

**Review by leogr — APPROVED** [review]

**Pull request merged**